### PR TITLE
align code with database and add local user for local development

### PR DESCRIPTION
**Pull request by teobler**

### CODE REVIEW

**Comment by teobler** [comment]
local user in memory will always be init value, cannot get the logined user

**Comment by teobler** [comment]
migration guide: https://planetscale.com/docs/tutorials/automatic-prisma-migrations

ref blog: https://www.nexxel.dev/blog/typesafe-database

File: src/pages/api/chatgpt/chat.ts [hunk 1]
```diff
@@ -1,22 +1,97 @@
...
+  }
+
+  if (req.method === "POST" && req.body) {
+    const {
+      prompt,
+      chat_id: chatId,
+      chat_name: chatName,
+    } = typeof req.body === "string" ? JSON.parse(req.body) : req.body;
+
+    if (prompt && chatId) {
```
Comment (phodal) [inline src/pages/api/chatgpt/chat.ts:40]: 这几个 if 可以重构一下，可以先 return 失败的场景

**Comment by phodal** [comment]
你的 PR 应该回不来了，要不再起个新的？

**Comment by teobler** [comment]
笑死，行

**Pull request closed without merging**